tests/ui/structopt_empty_attr.stderr

```diff
@@ -1,7 +1,5 @@
-error: proc-macro derive panicked
+error: unexpected end of input, expected parentheses
```

Comment:
Not possible to have span on the attribute here?

Reply:
Done. Not the cleanest code I've ever wrote, by our options are limited here by `trait syn::Parse` interaface